### PR TITLE
add travis lint script

### PR DESCRIPTION
**Pull request by 6kerokerotm9**

### CODE REVIEW

File: tools/setup.py [hunk 1]
```diff
@@ -52,11 +52,12 @@ def setup_rpc(self):
             devnull = open(os.devnull, 'wb')
             subprocess.check_call("git checkout master", stdout=devnull, stderr=subprocess.STDOUT)
             subprocess.check_call("git fetch origin", stdout=devnull, stderr=subprocess.STDOUT)
-            subprocess.check_call("git reset --hard origin/master", stdout=devnull, stderr=subprocess.STDOUT)
+            subprocess.check_call("git reset --hard origin/master",
```
Comment (jerrylee17) [inline tools/setup.py:55]: Does this compile? Generally when there's a line break you would have a `\` at the end of the line. 
`subprocess.check_call("git reset --hard origin/master", \`

File: tools/colors.py [hunk 2]
```diff
@@ -37,16 +43,34 @@ def print_color(self, color_type, string, is_input=False):
...
     def print_pink(self, string, is_input=False):
+        """
+        prints given text in pink
+        default treats input as user input
+        """
         self.print_color(self.PINK, string, is_input)
 
     def print_purple(self, string, is_input=False):
+        """
+        prints given text in pink
```
Comment (seemavora) [inline tools/colors.py:54]: not sure if u want to comment this every time, but should be purple here

File: tools/colors.py [hunk 3]
```diff
@@ -37,16 +43,34 @@ def print_color(self, color_type, string, is_input=False):
...
+        default treats input as user input
+        """
         self.print_color(self.PURPLE, string, is_input)
 
     def print_green(self, string, is_input=False):
+        """
+        prints given text in pink
```
Comment (seemavora) [inline tools/colors.py:61]: green

File: .pylintrc [hunk 4]
```diff
@@ -0,0 +1,597 @@
+[MASTER]
```
Comment (evanugarte) [inline .pylintrc:1]: if theres any lines that have nothing on the right side of the equals sign, remove it and the associated comments, lets make this file small

File: sce.py [hunk 5]
```diff
@@ -1,28 +1,14 @@
+"""
+This program uses setup.py to check if the user has the
```
Comment (evanugarte) [inline sce.py:2]: we dont need docstrings anymore so u can remove this

File: tools/colors.py [hunk 6]
```diff
@@ -11,9 +11,15 @@ class Colors:
     CLEAR = '\x1b[2J\x1b[H'
 
     def clear_screen(self):
+        """
```
Comment (evanugarte) [inline tools/colors.py:14]: remove all the docstrings from this file

File: tools/setup.py (hunk 1 above)
Comment (evanugarte) [reply tools/setup.py:55]: u only need that for strings, see the [pep8 rules](https://www.python.org/dev/peps/pep-0008/#indentation)

**Pull request merged**